src/qibolab/dummy.py

```diff
@@ -219,10 +307,15 @@ def create_dummy():
     channels |= (Channel(f"drive-{i}", port=instrument[f"drive-{i}"]) for i in range(nqubits))
     channels |= (Channel(f"flux-{i}", port=instrument[f"flux-{i}"]) for i in range(nqubits))
     channels |= Channel("twpa", port=None)
+    # FIXME: Issues with the names if they are strings maybe
+    channels |= (
+        Channel(f"flux_coupler-{c}", port=instrument[f"flux_coupler-{c}"])
```

Comment:
@stavros11 maybe it would be comfortable to have a factory for `Channel`s to avoid this repeated pattern:
```suggestion
    i1c = InstrumentChannel(instrument1)
    channels |= i1c(f"flux_coupler-{c}")
```

Reply:
Possibly convert to its own issue

Reply:
Ok, then let's just close :)